composeApp/build.gradle.kts

```diff
@@ -86,5 +85,18 @@ android {
     }
     dependencies {
         debugImplementation(compose.uiTooling)
+        implementation("androidx.compose.ui:ui:1.4.0")
+        implementation("androidx.compose.material:material:1.4.0")
+        implementation("androidx.compose.ui:ui-tooling-preview:1.4.0")
+        implementation("androidx.lifecycle:lifecycle-runtime-ktx:2.5.1")
+        implementation("androidx.activity:activity-compose:1.5.1")
+        implementation("cafe.adriel.voyager:voyager-navigator:1.0.0-rc01")
+        implementation("cafe.adriel.voyager:voyager-core:1.0.0-rc01")
+        implementation ("io.coil-kt:coil-compose:2.6.0")
+
+
```

Comment:
These dependencies shouldn't be here. Also, I am sure we already import them.. Check `commonMain.dependencies` and use the version catalog way of implementing dependencies.